examples/domain-generalization-with-kd/README.md

```diff
@@ -69,7 +72,8 @@ python primeqa/mrc/run_mrc.py \
        --overwrite_cache       
 ```
 
-We also provide a script for the conversion of MRQA 2019-style datasets (https://github.com/mrqa/MRQA-Shared-Task-2019) to the Hugging Face format:
+We also provide a script for the conversion of MRQA 2019-style datasets (https://github.com/mrqa/MRQA-Shared-Task-2019) to the Hugging Face Datasets SQuAD format:
 ```
-python src/convert_to_hf_format.py <path-to-input-mrqa-style-file> <path-to-output-hf-style-file>
+python utils/convert_to_hf_format.py <path-to-input-mrqa-style-file> <path-to-output-hf-style-file>
 ```
+The first argument is the path to the input input file (in MRQA format) and the second argument is the output file path which will contain the data in HF Datasets SQuAD-style format.
```

Comment:
The MRQA format is the same as the original SQuAD format right? @bhavani105 this script could help you - though I think it would be helpful to have a preprocessor in the original SQuAD format.

Reply:
@seirasto No, MRQA data files have a different format than the original SQuAD.

Reply:
I have an update to the SQuAD preprocessor that enables it to handle both original and HF formats. I'll create a PR